<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <resources><include location="../../../res/extui/extui.qrc"/></resources><author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>QG_InsertOptions</class>
  <widget class="QWidget" name="QG_InsertOptions">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>550</width>
        <height>24</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy>
        <hsizetype>4</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="minimumSize">
      <size>
        <width>550</width>
        <height>22</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>600</width>
        <height>32767</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Insert Options</string>
    </property>
    <property name="toolTip" stdset="0">
      <string/>
    </property>
    <layout class="QHBoxLayout">
      <property name="margin">
        <number>1</number>
      </property>
      <item>
        <widget class="QLabel" name="lAngle">
          <property name="text">
            <string>Angle:</string>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="leAngle">
          <property name="toolTip" stdset="0">
            <string>Rotation Angle</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="lFactor">
          <property name="text">
            <string>Factor:</string>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="leFactor">
          <property name="toolTip" stdset="0">
            <string>Scale Factor</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="Line" name="sep1_2">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape">
            <enum>QFrame::VLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="lArray">
          <property name="text">
            <string>Array:</string>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QSpinBox" name="sbColumns">
          <property name="maximum">
            <number>10000</number>
          </property>
          <property name="minimum">
            <number>1</number>
          </property>
          <property name="toolTip" stdset="0">
            <string>Number of Columns</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QSpinBox" name="sbRows">
          <property name="maximum">
            <number>10000</number>
          </property>
          <property name="minimum">
            <number>1</number>
          </property>
          <property name="toolTip" stdset="0">
            <string>Number of Rows</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="lSpacing">
          <property name="text">
            <string>Spacing:</string>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="leColumnSpacing">
          <property name="toolTip" stdset="0">
            <string>Column Spacing</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="leRowSpacing">
          <property name="toolTip" stdset="0">
            <string>Row Spacing</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="Line" name="sep1">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape">
            <enum>QFrame::VLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <connections>
    <connection>
      <sender>leAngle</sender>
      <signal>textChanged(QString)</signal>
      <receiver>QG_InsertOptions</receiver>
      <slot>updateData()</slot>
    </connection>
    <connection>
      <sender>leFactor</sender>
      <signal>textChanged(QString)</signal>
      <receiver>QG_InsertOptions</receiver>
      <slot>updateData()</slot>
    </connection>
    <connection>
      <sender>sbColumns</sender>
      <signal>valueChanged(int)</signal>
      <receiver>QG_InsertOptions</receiver>
      <slot>updateData()</slot>
    </connection>
    <connection>
      <sender>sbRows</sender>
      <signal>valueChanged(int)</signal>
      <receiver>QG_InsertOptions</receiver>
      <slot>updateData()</slot>
    </connection>
    <connection>
      <sender>leColumnSpacing</sender>
      <signal>textChanged(QString)</signal>
      <receiver>QG_InsertOptions</receiver>
      <slot>updateData()</slot>
    </connection>
    <connection>
      <sender>leRowSpacing</sender>
      <signal>textChanged(QString)</signal>
      <receiver>QG_InsertOptions</receiver>
      <slot>updateData()</slot>
    </connection>
  </connections>
</ui>
